src/mui/components/nested-dropdown/NestedDropdownItem.tsx

```diff
@@ -1,9 +1,15 @@
-import Container from "@mui/material/Container";
+import Box from "@mui/material/Box";
 import ListItemIcon from "@mui/material/ListItemIcon";
 import MenuItem from "@mui/material/MenuItem";
+import { styled } from "@mui/material/styles";
 import Typography, { TypographyProps } from "@mui/material/Typography";
 import React, { useCallback } from "react";
 
+const StyledListItemIcon = styled(ListItemIcon)({
+    marginLeft: "10px", // this is ridiculus, but the only way to align icons
+    marginRight: "-15px", // the only solution for now
+});
```

Comment:
Try to make it absolute positioned

Reply:
There's a default right margin of some set size for ItemListIcon, I couldn't find a way to adjust or access it. Only an override like this. 
With absolute position it still would have a margin.

Reply:
With the `position: absolute, right: 0`
![image](https://github.com/Exabyte-io/cove.js/assets/79542055/dd6a0f23-0223-46ef-b297-8ed9ea984eed)